bodhi/server/models.py

```diff
@@ -2998,7 +2998,7 @@ def obsolete_if_unstable(self, db):
         return
 
     def comment(self, session, text, karma=0, author=None, anonymous=False,
-                karma_critpath=0, bug_feedback=None, testcase_feedback=None,
+                karma_critpath=0, bug_feedback=[], testcase_feedback=[],
```

Comment:
In Python it's problematic to set function parameter defaults to mutable objects, because what you might expect isn't what happens. This bug_feedback list object will be instantiated once in the lifetime of the process and will persist between method calls. This means that if anything in the method ever alters the list, the altered list will be used as the default during the next method call. The technique of using ```None``` is a common way to avoid this problem.